doc/rst/usingchapel/chplenv.rst

```diff
@@ -508,6 +508,28 @@ CHPL_HWLOC
    to ``none`` and rebuild (and please file a bug with the Chapel team.) Note
    that building without hwloc will have a negative impact on performance.
 
+.. _readme-chplenv.CHPL_JEMALLOC:
+
+CHPL_JEMALLOC
+~~~~~~~~~~~~~
+   Optionally, the ``CHPL_JEMALLOC`` environment variable can select
+   between no jemalloc, using the jemalloc distributed with Chapel in
+   third-party, or using a system jemalloc. This setting is intented
+   to elaborate upon ``CHPL_MEM=jemalloc``.
+
+       ======== ==============================================================
+       Value    Description
+       ======== ==============================================================
+       none     do not build or use jemalloc
```

Comment:
Is CHPL_JEMALLOC even referenced if CHPL_MEM != jemalloc and, if so, for what purpose?  It seems like CHPL_JEMALLOC=none shouldn't be necessary.